python_modules/dagster/dagster/_core/execution/backfill.py

```diff
@@ -38,6 +40,13 @@ def from_graphql_input(graphql_str):
         return BulkActionStatus(graphql_str)
 
 
+@record
+class BulkActionsFilter:
+    status: Optional[BulkActionStatus] = None
```

Comment:
Probably we will need to expose most of the fields that we currently do on the RunsFilter.  Support for those fields is probably out of scope for this PR though.

The question about supporting single vs multiple statuses though will probably depend on the mapping of BulkActionStatus to the various types in the Runs view UI status filters.

We should flag that as a thing we need to figure out (status), as well as some of the other filters (e.g. job_name, tags, etc).

Reply:
yep - to add more context if someone ends up looking at this PR in the future: the main goal of adding this filter now is to remove this block of code https://github.com/dagster-io/dagster/pull/23560/files#diff-c46424223e58f81b69323dee30256f0a38733eac79198279ebbcf5cec19ee16eL461-L492 

I noted these things down for the filtering discussion